openpype/hosts/maya/plugins/load/load_multiverse_usd.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+import maya.cmds as cmds
+
+from openpype.pipeline import (
+    load,
+    get_representation_path
+)
+from openpype.hosts.maya.api.lib import (
+    maintained_selection,
+    namespaced,
+    unique_namespace
+)
+from openpype.hosts.maya.api.pipeline import containerise
+
+
+class MultiverseUsdLoader(load.LoaderPlugin):
+    """Load the USD by Multiverse"""
+
+    families = ["model", "usd", "usdComposition", "usdOverride"]
```

Comment:
Since you also allow `abc` content to be loaded I think  you might want to also add the `pointcache` and `animation` (output from a rig) families - they both produce Alembic files.

Reply:
Did in 5adf3966106f528bdfc38ccc2f7d1a1171efcb5b